app/src/main/java/com/rahobbs/todo/helpers/TodoItem.java

```diff
@@ -0,0 +1,88 @@
+package com.rahobbs.todo.helpers;
+
+import java.util.Date;
+import java.util.UUID;
+
+/**
+ * Created by rachael on 2/1/16.
+ * Model for a single to-do item
+ */
+public class TodoItem {
+    private String mTitle;
+    private String mDetails;
+    private Date mDate;
+    private UUID mID;
+    private boolean completed;
+    private String mParents;
+    private String mChildren;
+
+
+    public TodoItem() {
+        mID = UUID.randomUUID();
+        mDate = new Date();
+    }
+
+    public TodoItem(UUID id) {
+        mID = id;
+        mDate = new Date();
+    }
+
+    public boolean isCompleted() {
+
+        return completed;
+    }
+
+    public void setCompleted(boolean completed) {
+
```

Comment:
super nitpicky but there are soooo many extra spaces here. I would delete all of the spaces in the methods so they look like this:

```java
  public void setCompleted(boolean completed) {
    this.completed = completed;
  }
```